yt_dlp/extractor/recurbate.py

```diff
@@ -0,0 +1,43 @@
+from .common import InfoExtractor
+from ..utils import ExtractorError, merge_dicts
+
+
+class RecurbateIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?recurbate\.com/play\.php\?video=(?P<id>[0-9]+)'
+    _TESTS = [{
+        'url': 'https://recurbate.com/play.php?video=39161415',
+        'md5': 'dd2b4ec57aa3e3572cb5cf0997fca99f',
+        'info_dict': {
+            'id': '39161415',
+            'ext': 'mp4',
+            'description': 'md5:db48d09e4d93fc715f47fd3d6b7edd51',
+            'title': 'Performer zsnicole33 show on 2022-10-25 20:23, Chaturbate Archive – Recurbate',
+            'age_limit': 18,
+        },
+        'skip': 'Website require membership.',
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        try:
+            webpage = self._download_webpage(url, video_id)
+        except ExtractorError as e:
+            if e.cause.code == 403:
+                self.raise_login_required(method='cookies')
```

Comment:
`import urllib.error` above and:
```suggestion
            if isinstance(e.cause, urllib.error.HTTPError) and e.cause.code == 403:
                self.raise_login_required()
            raise
```